app/scripts/components/common/layout-root/index.tsx

```diff
@@ -1,14 +1,20 @@
-import React, { ReactNode, useContext, useCallback, useEffect } from 'react';
+import React, {
+  ReactNode,
+  useContext,
+  useCallback,
+  useEffect,
+  useState
+} from 'react';
 import { useDeepCompareEffect } from 'use-deep-compare';
 import styled from 'styled-components';
 import { Outlet } from 'react-router';
 import { reveal } from '@devseed-ui/animation';
-import { getCookieConsentFromVedaConfig } from 'veda';
+import { getBannerFromVedaConfig, getCookieConsentFromVedaConfig } from 'veda';
 import MetaTags from '../meta-tags';
 import PageFooter from '../page-footer';
 import Banner from '../banner';
 import { CookieConsent } from '../cookie-consent';
```

Comment:
Since we are on it, let's lazyload Banner and CookieConsent (the components that will render only when the configs are there)!